Beobachter.js

```diff
@@ -183,38 +107,67 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "http://www.beobachter.ch/natur/forschung-wissen/klima-wetter/artikel/blitzschlag_suche-nicht-die-buche/",
+		"url": "https://www.beobachter.ch/umwelt/blitze-suche-nicht-die-buche",
 		"items": [
 			{
 				"itemType": "magazineArticle",
+				"title": "Blitze: Suche nicht die Buche! | Beobachter",
```

Comment:
We should strip `| Beobachter` from the title, but otherwise this looks good!

Reply:
OK, should be all set